tests/tests/sorting.rs

```diff
@@ -0,0 +1,165 @@
+
+        CALL paradedb.create_bm25(
+            index_name => 'bm25_search',
+            table_name => 'bm25_search',
+            schema_name => 'paradedb',
+            key_field => 'id',
+            text_fields => paradedb.field('description') || paradedb.field('category', fast=>true, normalizer=>'raw'),
+            numeric_fields => paradedb.field('rating'),
+            boolean_fields => paradedb.field('in_stock'),
+            json_fields => paradedb.field('metadata'),
+            datetime_fields => paradedb.field('created_at') || paradedb.field('last_updated_date') || paradedb.field('latest_available_time')        
+        );        
+    "#.execute(&mut conn);
+
+    let (plan, ) = "EXPLAIN (ANALYZE, FORMAT JSON) SELECT * FROM paradedb.bm25_search WHERE description @@@ 'keyboard OR shoes' ORDER BY category LIMIT 5".fetch_one::<(Value,)>(&mut conn);
+    let plan = plan
+        .get(0)
+        .unwrap()
+        .as_object()
+        .unwrap()
+        .get("Plan")
+        .unwrap()
+        .as_object()
+        .unwrap()
+        .get("Plans")
+        .unwrap()
+        .as_array()
+        .unwrap()
+        .first()
+        .unwrap()
+        .as_object()
+        .unwrap();
+    eprintln!("{plan:#?}");
+    assert_eq!(
+        plan.get("   Sort Field"),
+        Some(&Value::String(String::from("category")))
+    );
+}
+
+#[rstest]
+fn sort_by_row_return_scores(mut conn: PgConnection) {
+    // ensure our custom scan wins against our small test table
+    r#"
+        SET enable_indexscan TO off;
+        CALL paradedb.create_bm25_test_table(table_name => 'bm25_search', schema_name => 'paradedb');
+
+        CALL paradedb.create_bm25(
+            index_name => 'bm25_search',
+            table_name => 'bm25_search',
+            schema_name => 'paradedb',
+            key_field => 'id',
+            text_fields => paradedb.field('description') || paradedb.field('category', fast=>true, normalizer=>'raw'),
+            numeric_fields => paradedb.field('rating'),
+            boolean_fields => paradedb.field('in_stock'),
+            json_fields => paradedb.field('metadata'),
+            datetime_fields => paradedb.field('created_at') || paradedb.field('last_updated_date') || paradedb.field('latest_available_time')        
+        );        
+    "#.execute(&mut conn);
+
+    let (plan, ) = "EXPLAIN (ANALYZE, FORMAT JSON) SELECT paradedb.score(id), * FROM paradedb.bm25_search WHERE description @@@ 'keyboard OR shoes' ORDER BY category LIMIT 5".fetch_one::<(Value,)>(&mut conn);
+    let plan = plan
+        .get(0)
+        .unwrap()
+        .as_object()
+        .unwrap()
+        .get("Plan")
+        .unwrap()
+        .as_object()
+        .unwrap()
+        .get("Plans")
+        .unwrap()
+        .as_array()
+        .unwrap()
+        .first()
+        .unwrap()
+        .as_object()
+        .unwrap()
+        .get("Plans")
+        .unwrap()
+        .as_array()
+        .unwrap()
+        .first()
```

Comment:
```suggestion
        .pointer("/0/Plan/Plans/0/Plans/0")
```

The `pointer` method from `serde_json` could be helpful here.